common/ManagementTestShared/ManagementRecordTestBase.cs

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
common/ManagementTestShared/ManagementRecordTestBase.cs is the old one, which we don't want to touch. 

How should we go about naming our new one as to distinguish it. ManagedSetupRecordBase? Something indicating it manages setup and teardown etc.?

Reply:
current it sounds like a "record" or a "data entry" like I am testing a management record.  Lets do some research on how many places this is used today and see if we can do an "update" on those to be compatible with our changes and have 1 class instead of 2 options

Reply:
updated

Reply:
Why not put this file into the main test framework?

Reply:
I agree that we shouldn't have these in the common directory like this unless there is a plan to be consumed from multiple projects and even in that case it might be better to keep them in the test framework directory and just be included from there as needed.

Reply:
This is still called RecordTestBase which still has the confusion I mentioned above

common/ManagementTestShared/ManagementRecordTestBase.cs
common/ManagementTestShared/ManagementRecordedTestBase.cs